Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "flate2"
 authors = ["Alex Crichton <alex@alexcrichton.com>", "Josh Triplett <josh@joshtriplett.org>"]
-version = "1.0.35"
+version = "1.0.36"
```

Comment:
Could we have a minor bump for this due to the MSRV update?

Reply:
sure, I've bumped it to `1.1.0`.